pkg/mount/mount.go

```diff
@@ -2,12 +2,48 @@ package mount // import "github.com/docker/docker/pkg/mount"
 
 import (
 	"sort"
+	"strconv"
 	"strings"
 	"syscall"
 
+	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
+// Error records an error from mount or unmount operation
+type Error struct {
+	op             string
+	source, target string
+	flags          uintptr
+	data           string
+	err            error
+}
+
+func (e *Error) Error() string {
+	out := e.op + " "
```

Comment:
Should we have the error start with "failed to " (e.g.)?

Reply:
Well, all the error messages (say from `os` package) are in the form `op: msg`, for example

`open: permission denied`

or

`readlink: no such file or directory`

I think we should follow the pattern. Brevity is the soul of wit :smile_cat:, besides, "failed to" is redundant here -- it's an error, so it is implied that something has failed.